content/articles/2020-07-24-On-the-move!.md

```diff
@@ -0,0 +1,25 @@
+Title: On the move!
+Date: 2020-07-24 09:00 CEST
+Tags: GSOC,GSOC20,Google,poliastro
+slug: 2020-07-24-On-the-move!
+lang: en
+Author: Meuge
+
+Hey, folks! Here's Meuge, your usual host from the last few months. 
+
+![Johnny](https://media.giphy.com/media/RoajqIorBfSE/giphy.gif)
+
+In the last few weeks, we got a little bit of everything, analyzing, coding, and searching for feasible solutions to reach the best result. Therefore, we came up with a new feature in Poliastro's Earth module. The API may change in the future, so be aware by the time you read this. Hence, I bet you might be wondering, what's all the fuzz?
+
+![Really?](https://media.giphy.com/media/US0y4X7FONHaM/giphy.gif)
+
+Well, now you can experiment with the new API to compute the visibility of sensors that are on board of the satellites. The FOV, field of view, the question is about meeting mission goals within the limit of a satellite sensor. 
+Given some parameters about the viewing satellite, the fundamental problem related to calculating the field of view is estimating how much of Earth we can see.
```

Comment:
A link to `sensors.py` (given that we don't have documentation yet) or even a very short code sample? :innocent: